.npmignore

```diff
@@ -0,0 +1,5 @@
+﻿node_modules
+package-lock.json
+test-results.xml
+test
```

Comment:
This keeps the tests out of the published package, making the published package smaller